src/preferences/effectmanifesttablemodel.cpp

```diff
@@ -0,0 +1,148 @@
+#include "preferences/effectmanifesttablemodel.h"
+
+#include <QMimeData>
+
+#include "effects/backends/effectsbackendmanager.h"
+
+namespace {
+const int kColumnType = 0;
+const int kColumnName = 1;
+const int kNumberOfColumns = 2;
+const QString kMimeTextDelimeter = "\n";
```

Comment:
```suggestion
const QString kMimeTextDelimeter = QStringLiteral("\n");
```

Reply:
fixed in 1b246e3f9a576de3934b831259adfb76806355f3